modular_zubbers/code/modules/cargo/expressconsole.dm

```diff
@@ -0,0 +1,30 @@
+/obj/item/circuitboard/computer/cargo/express/interdyne
+	name = "Interdyne Express Supply Console"
+	build_path = /obj/machinery/computer/cargo/express/interdyne
+
+/obj/machinery/computer/cargo/express/interdyne
+	name = "interdyne express supply console"
+	desc = "A standard NT express console, hacked by Gorlex industries to use \
+	their own experimental \"1100mm Rail Cannon\", made to be extra robust to prevent \
+	being emagged by the Syndicate cadets of the SSV Dauntless."
+	circuit = /obj/item/circuitboard/computer/cargo/express/interdyne
+	req_access = list(ACCESS_SYNDICATE)
+
+	podType = /obj/structure/closet/supplypod/bluespacepod
+
+	cargo_account = ACCOUNT_INT
+
+/obj/machinery/computer/cargo/express/interdyne/emag_act(mob/user, obj/item/card/emag/emag_card)
+	if(user)
+		if (emag_card)
+			user.visible_message(span_warning("[user] fails to swipe [emag_card] through [src]!"))
```

Comment:
Whilst nobody will probably ever see it, making a visible message on fail emag could be a bad idea.
```suggestion
```